### PR TITLE
feat: :sparkles: add command to make Python function file and it's test

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

This stacked PR moves the justfile recipe to make a Python file for a function and it's accompanying test file.

We could probably expand on this to also add template code too!

This PR needs a quick review.

## Checklist

- [x] Ran `just run-all`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +73,27 @@ This command only works if:
 
 - you are in a local Git repository
 - the repository has a file called ‘.pre-commit-config.yaml’
+
+``` bash
+spaid_create_function_file -h
```
Comment (signekb) [inline README.md:78]: Hmm, could it be 
```suggestion
spaid_create_python_file -h
```
instead? Just thinking with e.g., the properties example below that the file doesn't actually contain functions - it contains classes.

**Review by signekb — CHANGES_REQUESTED** [review]
It doesn't quite work as expected for me: It seems that no matter what I pass it (correct args, incorrect args (i.e., to non-existing folders), or no args), it creates the following two files in the root directory: `test_file` and `py_file`. 

Could I be doing something wrong? 
1. I went to my local `spaid` repo and pulled this branch 
2. I went into `seedcase_sprout` on the main branch 
3. In the terminal, I ran `spaid_setup_dev_workspace` 
4. Then, I ran `spaid_create_function_file` and gave it different args

File: bin/spaid_create_function_file [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+if [[ ! -f "$pyproject_file" ]]
+then
+    echo "Error: No '$pyproject_file' found. You need to be in a Python project."
+    exit 1
+fi
+
+py_file=./$1/$2/$3.py
+test_file=./tests/$2/test_$3.py
+touch py_file test_file
```
Comment (martonvago) [inline bin/spaid_create_function_file:35]:
```suggestion
touch $py_file $test_file
```
This should fix it?

**Review by martonvago — CHANGES_REQUESTED** [review]
Nice! Just the same problem as Signe said

File: README.md [hunk 3]
```diff
@@ -75,3 +73,27 @@ This command only works if:
...
+```
+
+Usage: spaid_create_function_file \[-h\]
+
+Run this script to create a new function file with the corresponding
+test file.
+
+Example:
+
+    $ spaid_create_function_file seedcase_sprout core properties
```
Comment (signekb) [inline README.md:88]: With the current folder structure in the `seedcase-sprout` repo, this should be:
```suggestion
    $ spaid_create_function_file sprout core properties
```
But maybe this is intentional as we have agreed to rename the `sprout` folder?

File: bin/spaid_create_function_file (hunk 2 above)
Comment (signekb) [reply bin/spaid_create_function_file:35]: Yep, that fixes it for me 👍

**Review by signekb — COMMENTED** [review]
If you give a non-existent folder - like I just tried with `seedcase_sprout` -  the function file is not created, but the message says it is: 
```
seedcase-sprout % spaid_create_function_file seedcase_sprout core signe
touch: ./seedcase_sprout/core/signe.py: No such file or directory
Created:
*** ./seedcase_sprout/core/signe.py ***
./tests/core/test_signe.py
```
(Asterisks added by me for emphasis) 

Should/could this be changed so only the actually created files are echoed as created?

File: README.md (hunk 1 above)
Comment (lwjohnst86) [reply README.md:78]: Good point!

File: README.md (hunk 3 above)
Comment (lwjohnst86) [reply README.md:88]: I did that intentionally since we will be renaming the folder.

File: bin/spaid_create_function_file (hunk 2 above)
Comment (lwjohnst86) [reply bin/spaid_create_function_file:35]: So weird, I thought I fixed that... :shrug:

**Comment by lwjohnst86** [comment]
Works for me now!

**Pull request merged**